OloEngine/src/Platform/Linux/LinuxServerConsole.cpp

```
@@ -0,0 +1,88 @@
+    AbortStatePtr Create()
+    {
+        auto state = std::unique_ptr<AbortState, AbortStateDeleter>(new AbortState());
+        if (::pipe(state->WakeupPipe) != 0)
+        {
+            OLO_CORE_ERROR("[ServerConsolePlatform] Failed to create wakeup pipe");
+            return nullptr;
+        }
+        return state;
+    }
+
+    bool WaitForStdin(AbortState& state)
```

Cppcheck (2.20.0)

[style] The function 'WaitForStdin' is never used.

(unusedFunction)